### PR TITLE
:sparkles: Forgotten password flow !

### PR DESCRIPTION
**Pull request by IncSow**

### CODE REVIEW

**Comment by IncSow** [comment]
Try it out on your side : )

File: front-end/src/pages/forgotten_password.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+      const response = await sendForgottenPassword(email)
+
+      if (response[0]) {
+        setErr(response[0])
+
+        return
+      }
+
+      router.back()
```
Comment (DenisMarine) [inline front-end/src/pages/forgotten_password.jsx:39]: router.push("/login") no ?

File: front-end/src/pages/user/update_password.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    .string()
+    .required(t("E.passwordError"))
+    .min(12, t("E.passwordLengthError"))
+    .matches(/[a-z]/, t("E.passwordLowerCaseError"))
+    .matches(/[A-Z]/, t("E.passwordUpperCaseError"))
+    .matches(/[0-9]/, t("E.passwordNumberError"))
+    .matches(/[@$!%*#?&]/, t("E.passwordSpecialCharError")),
+})
+
+const CreateAddress = (props) => {
```
Comment (DenisMarine) [inline front-end/src/pages/user/update_password.jsx:35]: not CreateAddress

**Review by DenisMarine — CHANGES_REQUESTED** [review]

File: front-end/src/pages/forgotten_password.jsx (hunk 1 above)
Comment (IncSow) [reply front-end/src/pages/forgotten_password.jsx:39]: Doesn't matter; we always come from login

**Review by DenisMarine — APPROVED** [review]

**Pull request merged**